language_specific/java/running_examples.rst

```diff
@@ -1,9 +1,22 @@
-Since third-party Java libraries generally don't have a standard place to install them, you will likely have to set the ``CLASSPATH`` variable appropriately to indicate that you wish to use the Xapian Java bindings.
+Since third-party Java libraries generally don't have a standard place
+to install them, you will likely have to set the ``CLASSPATH``
+variable appropriately to indicate that you wish to use the Xapian
+Java bindings.
 
-There are two parts to the bindings: a jarfile (``xapian.jar``) containing the Java classes, and the JNI library (``libxapian_jni.jnilib``) that connects them to Xapian itself. The easiest way to get this working is to copy those two files to the top-level directory of this repository. If you built your own Java bindings, the files will be in ``java/built`` in the bindings source code. Then you can use the following classpath::
+There are two parts to the bindings: a jarfile (``xapian.jar``)
+containing the Java classes, and the JNI library (such as
+``libxapian_jni.so`` on linux, or ``libxapian_jni.jnilib`` on macOS)
```

Comment:
Should probably capitalise "Linux"; "macOS" looks weird, but it appears Apple have renamed it (again), so that's actually correct (for now at least).